examples/pytorch-image-models/requirements.txt

```diff
@@ -1,2 +1 @@
-timm
```

Comment:
Wouldn't it be better to actually install it from pip and not from source to make sure we use a stable release? Or do you think we need to have the most recent changes?